tests/preprocessing/test_preprocessor_remapper.py

```diff
@@ -106,3 +138,37 @@ def test_transform_loss_mask(input_imputer, input_remapper) -> None:
     loss_mask_training = input_imputer.loss_mask_training
     loss_mask_training = input_remapper.transform_loss_mask(loss_mask_training)
     assert torch.allclose(loss_mask_training, expected_output)
+
+
+def test_remap_log1p(input_remapper_1d) -> None:
+    x = torch.Tensor([[1.0, 2.0, 3.0, 4.0, 150.0, 5.0], [6.0, 7.0, 8.0, 9.0, 201.0, 10.0]])
+    expected_output = torch.Tensor(
+        [
+            [1.0, 2.0, 3.0, 4.0, np.log1p(150.0), 5.0],
+            [6.0, 7.0, 8.0, 9.0, np.log1p(201.0), 10.0],
+        ]
```

Comment:
I would suggest to test for multiple monomappers applied and also on a forcing variable. 
```suggestion
            [1.0, 2.0, 3.0, np.sqrt(4.0), np.log1p(150.0), 5.0],
            [6.0, 7.0, 8.0, np.sqrt(9.0), np.log1p(201.0), 10.0],
```